web/client-api/src/main/java/io/deephaven/web/client/api/barrage/def/TableAttributesDefinition.java

```diff
@@ -29,9 +32,8 @@ public TableAttributesDefinition(Map<String, String> keys, Set<String> remaining
         if (map.containsKey(HIERARCHICAL_SOURCE_INFO_ATTRIBUTE + "." + HIERARCHICAL_COLUMN_NAME)) {
```

Comment:
No need to add the prefix again, it's already added:
```suggestion
        if (map.containsKey(HIERARCHICAL_COLUMN_NAME)) {
```